components/QuoteSymbol.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
naming question: Should we prefix/suffix the name for svg components?

Reply:
We could; as is, it is an actual component as well, so either way works. If anyone has any strong opinions we can do so in another PR.